test/jttt/Core/UI/CommandLineUITest.java

```diff
@@ -17,7 +17,8 @@
 
 public class CommandLineUITest {
 
-    private final int CVH_GAME_TYPE = 3;
+    private final int CVH_GAME_TYPE_OPTION = 3;
```

Comment:
And I think this is where you created a couple constants for these magic numbers, but they're only available from the test?